MonoGame.Framework/Graphics/GraphicsDevice.cs

```diff
@@ -307,12 +307,22 @@ public void Present()
             PlatformPresent();
         }
 
-        /*
+#if WINDOWS && DIRECTX
         public void Present(Rectangle? sourceRectangle, Rectangle? destinationRectangle, IntPtr overrideWindowHandle)
         {
-            throw new NotImplementedException();
+            // TODO: How to handle sourceRectangle and destinationRectangle?
+            SwapChainRenderTarget scrt;
+            if (!_overrideWindows.TryGetValue(overrideWindowHandle, out scrt))
+            {
+                scrt = new SwapChainRenderTarget(this, overrideWindowHandle, PresentationParameters.BackBufferWidth, PresentationParameters.BackBufferHeight);

```

Comment:
Yea actually just close this. While it could still be done if I added code to do a manual copy / render from one to the other, as it turns out I am no longer using / needing this method.